### DISCUSSION TITLE
How to map TModel to DateTimePoint?

### DISCUSSION THREAD

**Discussion by seikosantana** [Live-Charts/LiveCharts2#1283] [Q&A]
I'm trying to use series' `Mapping` but the `Coordinate` accepts `double`. How do I set the DateTime value for its X axis in series mapping?

Thanks in advance!

**Comment by beto-rodriguez** [comment]
Hello, use the DateTime.Ticks property, here is the source code of the DateTime point:

https://github.com/beto-rodriguez/LiveCharts2/blob/master/src/LiveChartsCore/Defaults/DateTimePoint.cs#L100
